content/english/linux-basics/activity-4.md

```diff
@@ -24,8 +25,43 @@ mkdir files
 
 As you can see, now when `ls -l` is run, an additional directory named `files` appears
 
+### What can you do with these files?
+
+One example of how you can manipulate a file or directory is by moving it to another location. This is accomplished by the move command, `mv`. Let's practice by moving the `Music` directory into the `Documents` directory. 
+
+Type in the command listed below:
+```
+mv Music/ Documents/
+```
+
+Now type in `ls -l` one more time and look at the output.
+
+![ls-l command](../images/04_mv_music_dir.png?classes=border,shadow)
+
+You should see that the directory `Music` no longer appears, because it was moved into the `Documents` directory.
+
+{{% notice tip %}}
+We can see that it moved into `Documents` by using the `ls` command with an added directory path.
+```
+ls -l Documents/
+```
+{{% /notice %}}
+
+Let's practice `mv` one more time by moving it back. 
+
+Try it:
+```
+mv Documents/Music/ .
+```
+
+This moves the `Music` directory from the `Documents` directory, back to our current directory, denoted by `.` The period, or dot, is used in commands to denote the current directory. Typing `ls -l` one more time shows `Music` has returned to this directory. 
+
+![ls-l command](../images/03_ls-l.png?classes=border,shadow)
+
+*Great job, you now know how to move files around. We will use this knowledge soon to recover from the hack.*
+
 ### What if I want to create a new file?
-To create a file we can use the `touch` command. The tough command creates an empty file with no data. 
+To create a file we can use the `touch` command. The touch command creates an empty file with no data. 
 
 Try the command below to create a file named `file1`:
 
```

Comment:
Below:

An empty file **doesn't** have much use, so **let's** add some data to it, like our name.